sdk/python/feast/infra/passthrough_provider.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, config: RepoConfig):
         self.repo_config = config
         self._offline_store = None
         self._online_store = None
+        self._document_store = None
```

Comment:
This is no longer necessary

Reply:
Thanks for addressing the issue. One question, though: what if we want to continue using Redis or any other online store for usual features, and use PG vector solely for embedding and search? Do we have the option to use the online store and the document store in the feature_store.yaml, both?

Reply:
That could get complicated but agreed it'd be good to add. I could imagine a Redis + another DB layer would be an obvious one.

Reply:
Thank you again, guys, for these amazing features. Yes, having multiple online stores will make it easier to use the right database layer for the appropriate use case! 🙌